perception/navigator_lidar_oa/src/ConnectedComponents.h

```diff
@@ -83,6 +83,9 @@ struct objectMessage
 	uint32_t pclInliers = 0;
 	geometry_msgs::Vector3 normal;
 	float minHeightFromLidar,maxHeightFromLidar;
+	bool current = false;
+	bool locked = false;
```

Comment:
Agreed and this was actually the original plan. I actually wrote this rosservice code before but it isn't part of this PR. We can certainly add it back.